src/sentry/integrations/middleware/hybrid_cloud/parser.py

```diff
@@ -53,22 +52,16 @@ def __init__(
         self.error = error
 
 
-class BaseRequestParser(abc.ABC):
+class BaseRequestParser:
     """Base Class for Integration Request Parsers"""
 
     _METRIC_SUCCESS_KEY = "integrations.middleware.request_parser.success"
     _METRIC_FAILURE_KEY = "integrations.middleware.request_parser.failure"
     _METRICS_INFO_KEY = "integrations.middleware.request_parser.info"
 
-    @property
-    def provider(self) -> str:
```

Comment:
mypy (correctly) sees this as an instance property when the intent was classlevel based on usage and implementation